system/dd/dd_main.c

```diff
@@ -31,7 +31,9 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
+#if defined(__NuttX__)
```

Comment:
```suggestion
#ifdef __NuttX__
```

Reply:
are you suggesting to change all of them, including the existing one?

Reply:
done